atomicapp/external/templates/nulecule/answers.conf.sample.tpl

```diff
@@ -0,0 +1,7 @@
+[general]
+namespace = default
+provider = kubernetes
+
+[${app_name}]
+image = centos/httpd
+
```

Comment:
as what @dustymabe said, why do we have a answers.conf.sample.tpl template? we should be auto-generating this, not using a template.